ciao-launcher/payload.go

```diff
@@ -168,11 +170,13 @@ func parseStartPayload(data []byte) (*vmConfig, *payloadError) {
 	vnicIP := strings.TrimSpace(net.PrivateIP)
 	sshPort := computeSSHPort(networkNode, vnicIP)
 	var volumes []volumeConfig
-	if start.Storage.ID != "" {
-		volumes = append(volumes, volumeConfig{
-			UUID:     start.Storage.ID,
-			Bootable: start.Storage.Bootable,
-		})
+	for _, storage := range start.Storage {
```

Comment:
storage.ID == "" would be the case for a "local" volume, which launcher would auto-create/destroy, but we've discussed not supporting that.  I'll add a check for that and open an issue for tracking eventual support.